dotnet/src/Extensions/Planning.StepwisePlanner/Skills/StepwiseStep/config.json

```diff
@@ -12,20 +12,15 @@
   },
   "input": {
     "parameters": [
-      {
-        "name": "question",
-        "description": "The question to answer",
-        "defaultValue": ""
-      },
-      {
-        "name": "agentScratchPad",
-        "description": "The agent's scratch pad",
-        "defaultValue": ""
-      },
       {
         "name": "functionDescriptions",
         "description": "The manual of the agent's functions",
         "defaultValue": ""
+      },
+      {
+        "name": "suffix",
+        "description": "",
+        "defaultValue": "Let's break down the problem step by step and think about the best approach. Label steps as they are taken.\n\nContinue the thought process!"
```

Comment:
This is important to include even if the user wants to provide supplemental usage instructions. Otherwise, it'd be overridden and there'd be no pointers to go step by step

Reply:
Yeah, in the example usage I'm building on this value in the configuration. Trying to find the balance between customization and guardrails.

Reply:
e.g. `plannerConfig.Suffix = $"{Suffix}\n{plannerConfig.Suffix}";`